### PR TITLE
Create report for variable gas usage when updating drip and fetching chi

### PR DESCRIPTION
**Pull request by epheph**
If we went with 1 year, any request that
 - contains "updateAndFetchChi" = +50,340
 - contains calcualteChi = +5,258 
 - is withdrawTo = 51,771

All the values below include the base transaction cost, but since we're only looking for difference between cached and non-cached run, base cost cancels out

There is one odd case, withdrawTo, where it does a now-protected drip, but doesn't fetch chi if rho == now. It's the only case where we want to drip, but don't need chi, so it's delta will be slightly larger

```
withdrawTo()
216102 gas : 0 minutes
263927 gas : 1 minutes
264959 gas : 60 minutes
265932 gas : 1440 minutes
267873 gas : 525600 minutes
268345 gas : 5256000 minutes

updateAndFetchChi()
25781 gas : 0 minutes
72112 gas : 1 minutes
73160 gas : 60 minutes
74149 gas : 1440 minutes
76121 gas : 525600 minutes
76590 gas : 5256000 minutes

calculateChi()
28208 gas : 0 minutes
29520 gas : 1 minutes
30552 gas : 60 minutes
31525 gas : 1440 minutes
33466 gas : 525600 minutes
33928 gas : 5256000 minutes
```

### CODE REVIEW

File: test/test.ts [hunk 1]
```diff
@@ -15,6 +15,7 @@ import { Actor, duplicateActor } from '../scripts/libraries/actor';
 import { TestDependencies } from './test-dependencies';
 import { generateDai } from '../scripts/seed/seed-maker';
 import { resetMaker } from '../scripts/seed/reset-maker';
+import { IOffChainTransaction } from "@zoltu/ethereum-types/output-node";
```
Comment (MicahZoltu) [inline test/test.ts]: This should be `import { IOffChainTransaction } from '@zoltu/ethereum-types'`.

Comment (epheph) [reply test/test.ts]: ahh, IDE burned me

**Comment by epheph** [comment]
New report
```
withdrawTo()
216102 gas : 0 seconds
262625 gas : 1 seconds
264959 gas : 3600 seconds
265932 gas : 86400 seconds
267873 gas : 31536000 seconds
268335 gas : 315360000 seconds
269465 gas : 33554431 seconds
270320 gas : 268435455 seconds

calculateChi()
28218 gas : 0 seconds
28218 gas : 1 seconds
30552 gas : 3600 seconds
31525 gas : 86400 seconds
33466 gas : 31536000 seconds
33928 gas : 315360000 seconds
35058 gas : 33554431 seconds
35913 gas : 268435455 seconds

updateAndFetchChi()
25781 gas : 0 seconds
70789 gas : 1 seconds
73160 gas : 3600 seconds
74149 gas : 86400 seconds
76121 gas : 31536000 seconds
76590 gas : 315360000 seconds
77738 gas : 33554431 seconds
78606 gas : 268435455 seconds

```

File: test/test.ts (hunk 1 above)
Comment (MicahZoltu) [reply test/test.ts]: Yeah, it does that to me sometimes to, I don't know why.

**Pull request merged**